### ISSUE TITLE
Can't go into spotify settings or change any window [DribbblishDynamic]

### ISSUE THREAD

**Issue by federicotorrielli** [morpheusthewhite/spicetify-themes#336]
**Describe the bug**
Video: https://streamable.com/xdhpwj

**To Reproduce**
Everything is shown in the video.

**Logs**
I can enable devtools but the devtools window does not show up

**Specifics (please complete the following information):**
 - OS: Arch Linux (latest)
 - Spotify version: 1:1.1.56.595-1
 - Spicetify version: 1.2.1
 - Theme name: DribbblishDynamic

**Comment by JulienMaille** [comment]
Sorry can't do anything without logs. Also I switched to the modern interface which is unfortunately not supported by spicetify.

**Comment by morpheusthewhite** [comment]
@federicotorrielli after running `spicetify enable-devtools` and restarting Spotify you should be able to right-click in the Spotify window and open the developer console

**Comment by briangzimmerman** [comment]
For what it's worth, I am also having this problem and I also cannot get the devtool window to show up. I ran `spicetify enable-devtool`, but right-clicking didn't do anything special and `crtl+shift+i` didn't do anything. I just started messing around with spicetify today, so I'm not very familiar with it. Maybe there's something I'm missing?

**Specifics (please complete the following information):**
 - OS: Fedora 33
 - Spotify version: 1.1.55.498.gf9a83c60 (Flatpak)
 - Spicetify version: 1.2.1
 - Theme name: DribbblishDynamic (Dark)

**Comment by JulienMaille** [comment]
off topic question: why are there so many linux users? Is spicetify promoted on a linux oriented website/community?

**Comment by morpheusthewhite** [comment]
I think because one of the main ways people learn about Spicetify is r/unixporn

**Comment by briangzimmerman** [comment]
Yeah, r/unixporn is where I found out about it.

About this issue: I figured out that I have to double right-click for the menu with the DevTools option to appear. There are no console errors that seem related to this issue. I dug around the `zlink.bundle.js` file a bit and it seems like the `menu-active-item` event is not always firing for some reason. I'm using spicetify on a laptop and I tried clicking on the settings option in 3 ways: by clicking with a bluetooth mouse, by tapping the touch pad, and by physically clicking the touch pad.  I'm really not sure why, but clicking the touch pad would work a little more often than tapping the touch pad or clicking the bluetooth mouse. It doesn't make much sense. Something else that would help is clicking the "About" option first. After I clicked "About," the "Settings" option would work.

There is consistently 1 console error when I open spotify, but I'm not sure it's related to this issue:
```
Uncaught TypeError: Cannot read property 'length' of undefined
    at zlink.bundle.js:495
    at zlink.bundle.js:495
    at zlink.bundle.js:495
    at zlink.bundle.js:495
    at zlink.bundle.js:495
```
